appveyor.yml

```diff
@@ -0,0 +1,41 @@
+version: "{build}"
+
+# Init is run before the repo is cloned
+# Windows line endings.
+init:
```

Comment:
Yeah you're right; I was new to this, so I had comments to remind myself. I'll remove the unnecessary comments.